### ISSUE TITLE
Invitation email received (gmail)

### ISSUE THREAD

**Issue by antobenve** [unep-grid/map-x-mgl#633]
The invitation email that is received by gmail users is badly formatted so that breaklines are not respected. This is the result. As gmail is quite a common mail provider, it might be a good idea to find a solution. Maybe a space before the breakline would make it already nicer without having a negative impact on the other users.
![image](https://user-images.githubusercontent.com/30286565/104213361-7acaa480-5436-11eb-9c12-a3a77a6c01d9.png)

Version 1.8.0-alpha using Chrome 87.0.4280.88 on Windows 10

**Comment by thomaspiller** [comment]
![image](https://user-images.githubusercontent.com/19908801/105187674-56a44d00-5b33-11eb-9f69-2fe12a25f97d.png)
Fixed in 1.8.4-beta. Thanks!

**Comment by thomaspiller** [comment]
![image](https://user-images.githubusercontent.com/19908801/105188139-dd592a00-5b33-11eb-8879-11dccca25883.png)
Not possible to have line breaks?

**Comment by antobenve** [comment]
That's exactly the issue. Until we find a solution the changes in dictionary will allow a reasonable spacing between sentences.

**Comment by fxi** [comment]
The email should render correctly on most platform.. \n should translate to </br> in email renderer.

![image](https://user-images.githubusercontent.com/1196833/105189212-04642b80-5b35-11eb-9cb9-acb1291c3721.png)

If it's not the case, I can do that in the API.

**Comment by fxi** [comment]
hee, this is already done...
```js
{ from: '<my email>',
  to: '<someone else email>',
  content: 'Hi! </br>I would like to invite you as a member to…</a> </br> The administrator of the project.</br>',
  title: 'MapX invitation',
  subtitle: 'Invitation to join the project "..."',
  ... }
```
What email client are you using ?

**Comment by thomaspiller** [comment]
Gmail client directly in the browser.

**Comment by fxi** [comment]
Ok. I got this : `</br>` instead of `<br>`...

**Comment by thomaspiller** [comment]
The problem is fixed in v1.8.4-beta.

![image](https://user-images.githubusercontent.com/19908801/105196753-3c22a180-5b3c-11eb-841a-4b3699a7acd1.png)
